samples/istio/quickstart/B.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  labels:
```

Comment:
The labels section (which looks like a remnant from `iter8-istio`) seems to add lines to this VS without any purpose.